prac_06/guitars.py

```diff
@@ -0,0 +1,18 @@
+from prac_06.guitar import Guitar
+
+my_guitars = []
+print("My Guitars!")
+guitar_name = input("Name: ")
+while guitar_name != "":
+    year = int(input("Year: ")) # Needs to be and int as its subtracted from CURRENT_YEAR in is_vintage()
```

Comment:
Good comment. 
Year must be an int, could consider error-checking or try-except to enforce?